packages/components/data-table/src/data-table.styles.js

```diff
@@ -17,6 +17,8 @@ const getClickableRowStyle = (props) => {
 };
 
 const TableContainer = styled.div`
+  overflow-x: auto;
```

Comment:
Because we added this container (to include the footer on the maxWidth constraint), we must also enable the horizontal scrollbars here.